C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -1644,16 +1659,13 @@ vector<string>  asset_codes;
 					sql.append(' ');
 				}
 
-				const char *sql_cmd = R"(
-						id,
-						asset_code,
-						reading,
-						strftime(')" F_DATEH24_SEC R"(', user_ts, 'localtime')  ||
-						substr(user_ts, instr(user_ts, '.'), 7) AS user_ts,
-						strftime(')" F_DATEH24_MS R"(', ts, 'localtime') AS ts
-                    FROM  )";
-
-				sql.append(sql_cmd);
+				sql.append("id, asset_code, reading, strftime(')" F_DATEH24_SEC);
+				sql.append("(', user_ts, '");
+				sql.append(timezone);
+				sql.append("')  || substr(user_ts, instr(user_ts, '.'), 7) AS user_ts, strftime(')" F_DATEH24_MS);
+			       	sql.append("(', ts, '");
+				sql.append(timezone);
+				sql.append("') AS ts FROM  )");
```

Comment:
Syntax error here; No need to close the bracket after FROM. Fixed in https://github.com/fledge-iot/fledge/pull/1118